builder/src/scss/all.scss

```diff
@@ -1,16 +1,16 @@
 // Bootstrap and theme
 @import "./theme/_variables";
-@import "node_modules/bootstrap/scss/bootstrap";
+@import "./node_modules/bootstrap/scss/bootstrap";
 @import "./theme/_bootswatch";
 
 // Font awesome
-@import "./vendor/fontawesome";
+@import "./node_modules/@fortawesome/fontawesome-free/css/all";
```

Comment:
It's unclear which version of fontawesome was used before. The list of fontawesome's CSS files:
```
73625 all.css
  732 brands.css
71990 fontawesome.css
  734 regular.css
  727 solid.css
  669 solid.min.css
 8077 svg-with-js.css
41312 v4-shims.css
```